README.md

```diff
@@ -109,20 +109,19 @@ For detailed steps on the installation, follow the guides below:
 - [How to build on Windows](https://github.com/0chain/zwalletcli/wiki/Build-Windows)
 
 ### 2. Run `zwallet` commands
-
-The following steps assume that your terminal's working directory is inside the `zwalletcli` repo.
-
+Ensure your terminal's working directory is inside the `zwalletcli` repo for the following steps.
+... MINGW64 ~/zwalletcli
 ## Global parameters
 
 `zwallet` accept global parameters to override default configuration and can be used in any command.
 
 | Parameter     | Description                     | Default        |
 | ------------- | ------------------------------- | -------------- |
-| `--h,--help`  | Show help/parameters for a particular command                       |                |
-| `--config`    | [Config file](https://github.com/0chain/zwalletcli/blob/staging/network/config.yaml) and [description](https://github.com/0chain/zwalletcli#zcnconfigyaml)   | `config.yaml`  |
-| `--configDir` | Config directory                | `~/.zcn`       |
-| `--network`   | [Network file](#override-network) | `network.yaml` |
-| `--silent`    | Do not print detailed logs      | `false`        |
+| `--h,--help`  | Shows help/parameters for a particular command (./zwallet -h) will list all help commands. To know detail about a particular command like example 'send' use (./zwallet send --help )                     |                |
+| `--config`    |  [Config file](https://github.com/0chain/zwalletcli/blob/staging/network/config.yaml) and [description](https://github.com/0chain/zwalletcli#zcnconfigyaml). Configuration file    | `config.yaml`  |
+| `--configDir` | Configuration directory                | `~/.zcn`       |
```

Comment:
incrase the parameter coplumn size so that commands can fit in and does not break line for commands